src/pg_to_evalscript/javascript_processes/mod.js

```diff
@@ -0,0 +1,29 @@
+function mod(arguments) {
+  const { x, y } = arguments;
+
+  if (x === undefined) {
+    throw new Error("Mandatory argument `x` is not defined.");
+  }
+
+  if (y === undefined) {
+    throw new Error("Mandatory argument `y` is not defined.");
+  }
+
+  if (y === 0) {
+    throw new Error("Division by zero is not supported.");
+  }
+
+  if (x === null || y === null) {
+    return null;
+  }
+
+  if (typeof x !== "number") {
+    throw new Error("Argument `x` is not a number.");
+  }
+
+  if (typeof y !== "number") {
+    throw new Error("Argument `y` is not a number.");
+  }
+
+  return x % y;
```

Comment:
From the [docs](https://docs.openeo.cloud/processes/#mod)
> The result of a modulo operation has the sign of the divisor.

Be sure to add the sign of the divisor (`y`) to the result. 

For example, `3 % -2` returns `1` in javascript.

Reply:
Added comments in the `test_mod.py` for convenience

Reply:
Should be fixed